packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -57,6 +58,11 @@ class WidgetTester extends Instrumentation {
     );
     async.flushMicrotasks();
   }
+
+  void dispatchRawEvent(PointerEvent event, HitTestResult result) {
```

Comment:
Rather than adding this, I'd remove the old dispatchEvent, which is deprecated and wrong.  Then we can just call this function dispatchEvent.